apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -203,145 +216,135 @@ function ListBoxInline({ options, layout }) {
+        onKeyDown={handleKeyDown}
+        onMouseEnter={handleOnMouseEnter}
+        onMouseLeave={handleOnMouseLeave}
+        ref={containerRef}
+      >
+        {toolbar && (
+          <Grid
+            item
+            container
+            style={{ padding: theme.spacing(1), paddingLeft: `${headerPaddingLeft}px` }}
+            wrap="nowrap"
+          >
+            <Grid item container height={headerHeight} wrap="nowrap">
+              {showIcons && (
+                <Grid item sx={{ display: 'flex', alignItems: 'center', width: iconsWidth }}>
+                  {isLocked ? (
+                    <IconButton tabIndex={-1} onClick={unlock} disabled={selectDisabled()} size="large">
+                      <Lock title={translator.get('Listbox.Unlock')} style={{ fontSize: '12px' }} />
+                    </IconButton>
+                  ) : (
+                    showSearchIcon && (
+                      <IconButton
+                        onClick={onShowSearch}
+                        tabIndex={-1}
+                        title={translator.get('Listbox.Search')}
+                        size="large"
+                      >
+                        <SearchIcon style={{ fontSize: '12px' }} />
+                      </IconButton>
+                    )
+                  )}
+                  {isDrillDown && (
                     <IconButton
-                      onClick={onShowSearch}
                       tabIndex={-1}
-                      title={translator.get('Listbox.Search')}
+                      title={translator.get('Listbox.DrillDown')}
                       size="large"
+                      sx={{ paddingLeft: `${drillDownPaddingLeft}px` }}
                     >
-                      <SearchIcon style={{ fontSize: '12px' }} />
+                      <DrillDownIcon style={{ fontSize: '12px' }} />
                     </IconButton>
-                  )
-                )}
-                {isDrillDown && (
-                  <IconButton
-                    tabIndex={-1}
-                    title={translator.get('Listbox.DrillDown')}
-                    size="large"
-                    sx={{ paddingLeft: `${drillDownPaddingLeft}px` }}
-                  >
-                    <DrillDownIcon style={{ fontSize: '12px' }} />
-                  </IconButton>
+                  )}
+                </Grid>
+              )}
+              <Grid item className={classes.listBoxHeader}>
+                {showTitle && (
+                  <Title variant="h6" noWrap ref={titleRef}>
+                    {layout.title}
+                  </Title>
                 )}
               </Grid>
-            )}
-            <Grid item className={classes.listBoxHeader}>
-              {showTitle && (
-                <Title variant="h6" noWrap>
-                  {layout.title}
-                </Title>
-              )}
+            </Grid>
+            <Grid item xs />
```

Comment:
What is this empty item doing here?

Reply:
Seems to have been there since this file was created, if removed the align of search glass is broken.

Reply:
😮 🆗